curator-operator-system/base/rbac.authorization.k8s.io/rolebindings/rolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: leader-election-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: leader-election-role
+subjects:
+- kind: ServiceAccount
```

Comment:
Looks like the serviceAccount isn't currently deployed in the curator-operator-system namespace, you will have to add it under apps/curator-operator-system/base

Reply:
Okay. I was creating a service account while deploying the project.

Reply:
Added service account.